lib/config/options/index.ts

```diff
@@ -648,6 +649,13 @@ const options: RenovateOptions[] = [
     default: true,
     supportedManagers: ['npm', 'composer'],
   },
+  {
+    name: 'issueCollector',
```

Comment:
Any recommendations on the new global configuration name to specify the issue collector?  Also thought about `issuePlatform`